From b5f71cefda98277272d483dd4a31c71208dafd5f Mon Sep 17 00:00:00 2001
From: "Zeeshan Ali (Khattak)" <zeeshanak@gnome.org>
Date: Thu, 6 Nov 2014 20:21:50 +0000
Subject: [PATCH 2/7] wifi: Remove a redundant condition

We were only refreshing the location on starting of wifi source if
requested accuracy level was 'city'. This resulted in clients requesting
accuracy level other than 'city' to not get any location on machines
without a wifi card (e.g VMs).

This patches fixes the issue by removing the redundant condition.
---
 src/gclue-wifi.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/gclue-wifi.c b/src/gclue-wifi.c
index cf2c455..e94ef04 100644
--- a/src/gclue-wifi.c
+++ b/src/gclue-wifi.c
@@ -377,8 +377,7 @@ connect_bss_signals (GClueWifi *wifi)
         if (priv->bss_added_id != 0)
                 return;
         if (priv->interface == NULL) {
-                if (wifi->priv->accuracy_level == GCLUE_ACCURACY_LEVEL_CITY)
-                        gclue_web_source_refresh (GCLUE_WEB_SOURCE (wifi));
+                gclue_web_source_refresh (GCLUE_WEB_SOURCE (wifi));
 
                 return;
         }
-- 
2.1.0

